playbooks/archivematica/Vagrantfile

```diff
@@ -39,7 +39,6 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     ansible.host_key_checking = false
     ansible.extra_vars = {
       "archivematica_src_dir" => "/vagrant/src",
-      "archivematica_src_environment_type" => "development",
```

Comment:
I think this line could be left in.  The Vagrantfile is meant to set up a development environment.  Maybe this file should override archivematica_src_am_version and archivematica_src_ss_version as well, to point at qa branches?  

The readme.md in this repo could be clear that 'vagrant up' will create a dev environment, and for production uses it is necessary to have a hosts file and do ansible-playbook commands instead.  When you use ansible-playbook, the Vagrantfile is ignored, and the extra_vars in there are not used.